### ISSUE TITLE
Can't open any menu while using the OpenGL renderer unless single-window mode is enabled (multi-window support not implemented yet in OpenGL)

### ISSUE THREAD

**Issue by himaghnam** [godotengine/godot#54795]
### Godot version

4.0 master 

### System information

windows 10, gles3 

### Issue description

Can't open the menu while in gles3 mode

### Steps to reproduce

just change the driver to gles3

**Comment by bruvzg** [comment]
I can reproduce this on Windows 11 (and Wine), but not on macOS, seems like platform specific issue.

Wine is prints the following error when trying to open a popup window:
```
wglSwapBuffers current context 0xc2e330 doesn't match hdc 0x3f0050; can't swap
```

**Comment by Calinou** [comment]
Support for multiple windows isn't implemented yet in the OpenGL renderer. As a workaround, enable **Single Window Mode** in the Editor Settings or use the `--single-window` command line argument.

**Comment by lawnjelly** [comment]
This may have been fixed by #65727 (I'm presuming GL uses multi window default?).

**Comment by akien-mga** [comment]
Seems to work fine in 4.0 alpha 17 indeed, tested on Linux with OpenGL3.
